server/libmain/libmain.go

```diff
@@ -62,7 +62,7 @@ var (
 	sslPort        = flag.Int("ssl_port", 8081, "The port to listen for HTTPS traffic on")
 	gRPCPort       = flag.Int("grpc_port", 1985, "The port to listen for gRPC traffic on")
 	gRPCSPort      = flag.Int("grpcs_port", 1986, "The port to listen for gRPCS traffic on")
-	monitoringPort = flag.Int("monitoring_port", 9090, "The port to listen for monitoring traffic on")
+	monitoringPort = flag.Int("monitoring_port", 9100, "The port to listen for monitoring traffic on")
```

Comment:
why change the default? is this gunna screw things up for folks already using these metrics?

Reply:
Hm, yeah if people are using the default value for the port then this will mess them up. Reverted this change and updated prometheus server configs to listen on a different port than 9090 so it doesn't conflict with BuildBuddy.